### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by hwangswan**
# Code refactor
## 1. Description
Code refactor.

## 2. List of issues fixed:
None

Details:
- [x] Update type hint for functions/methods.

Mentioning @khongsomeo/as101 for review.

### CODE REVIEW

File: src/exception.py [hunk 1]
```diff
@@ -22,11 +22,11 @@ class HTTPException(Exception):
     # Response from the request
     response: Any
 
-    def __init__(self, message: Optional[str] = None, **kwargs) -> str:
-        """Initialization
+    def __init__(self, message: Optional[str] = None, **kwargs):
```
Comment (trhgquan) [inline src/exception.py:25]: Should return None.

**Review by trhgquan — CHANGES_REQUESTED** [review]
Some changes requested.

File: src/exception.py (hunk 1 above)
Comment (hwangswan) [reply src/exception.py:25]: Affirmative.

**Review by trhgquan — APPROVED** [review]
**LGTM!**

**Pull request merged**